homeassistant/components/tuya/binary_sensor.py

```diff
@@ -261,11 +260,22 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
     # https://developer.tuya.com/en/docs/iot/categorysos?id=Kaiuz3oi6agjy
     "sos": (
         TuyaBinarySensorEntityDescription(
-            key=DPCode.SOS_STATE,
+            key=DPCode.BASIC_INDICATOR,
```

Comment:
This change seems incorrect? According to specifications, this is `sos_state`, see: 

<https://developer.tuya.com/en/docs/iot/s?id=K9gf48qoksops>